### ISSUE TITLE
Godot appears load error in macOS 14.5 

### ISSUE THREAD

**Issue by weiwangorg** [P1X-in/tanks-of-freedom-ii#27]
when loading master.zip in Godot 4.2, it appeared load errors following:
<img width="864" alt="image" src="https://github.com/P1X-in/tanks-of-freedom-ii/assets/22360336/bdaf568d-bc81-4d2f-95d9-b18bbbe65fe2">

**Comment by czlowiekimadlo** [comment]
Probably caused by this bug in Godot Engine: https://github.com/godotengine/godot/issues/81555

**Comment by GaidamakUA** [comment]
Yes, it's broken for me too. Part of the reason I want to migrate to 4.4
